### DISCUSSION TITLE
I hope not to block the web page right button

### DISCUSSION THREAD

**Discussion by cy-6359** [rocksdanister/lively#976] [Ideas]
Hello, developer, I'm a user from China. I'm a web developer. I made a navigation page and hope to make it into desktop wallpaper to improve my efficiency. However, the right-click menu of the web page I developed is customized. Many functions need to be called out in my customized menu. I tried many software. Only this compatibility is good, But I can't input Chinese and the right button of the outgoing page in the input box under the desktop. I hope you can liberate the right button function of the page in the future version. Thank you very much!

**Comment by rocksdanister** [comment]
Right button is disabled because Windows desktop already has a right-button menu:
<img width="266" alt="Screenshot 2022-01-02 183553" src="https://user-images.githubusercontent.com/17554161/147876724-29ca01cc-ee75-4e50-b21a-8120ae651b59.png">

**Comment by rocksdanister** [comment]
Maybe you can use Lively Properties API instead:
https://github.com/rocksdanister/lively/wiki/Web-Guide-IV-:-Interaction#lively-properties

**Reply by cy-6359** [reply]
Thank you for your reply. I have to study it for a while... Another small problem is that the input box cannot input Chinese。
https://img.wenhairu.com/images/2022/01/02/nQK7o.png

**Comment by rocksdanister** [comment]
Yes, its a known issue https://github.com/rocksdanister/lively/issues/316
